### ISSUE TITLE
Easier to read fonts

### ISSUE THREAD

**Issue by Singh-in** [friendica/friendica#9769]
### Is the feature request related to a problem? Please describe.

no Problem

### Describe the feature you'd like

The typeface of e.g. Mastodon and misskey is much clearer, more pleasant to read and also feels much easier on the eyes. Hence the question: would it be a good idea to achieve an alignment here?

### Describe alternatives you've considered

no

### Additional context

**Comment by MrPetovan** [comment]
The frio theme is using [Open Sans](https://www.opensans.com), do you happen to know what font Misskey and Mastodon are using?

**Comment by tobiasd** [comment]
From the CSS files Mastodon is using, it looks like [roboto](https://en.wikipedia.org/wiki/Roboto) disguised as mastodon-sans-serif or sans-serif for those who disabled embedded fonts.

**Comment by Singh-in** [comment]
I could not find any reference in the documentation. However, I keep coming across Roboto as a font

`@import "https://fonts.googleapis.com/css?family=Roboto:400,500,700|Roboto+Mono:400";/*!* Font Awesome Free 5.1.0 by @fontawesome - https://fontawesome.com`

**Comment by MrPetovan** [comment]
I do not mind changing the font to Roboto as the license is equally permissive, but I do feel like it's a purely aesthetic question we should submit to the community. [The Popular Pairings section of the official Roboto font page](https://fonts.google.com/specimen/Roboto#pairings) allows to quickly switch between Roboto and Open Sans and I must admit I personally prefer Open Sans for its greater width, both for headings and regular text. Roboto is a more compact and dense font and it feels more cramped to me when directly comparing with Open Sans.

**Comment by Extarys** [comment]
Without changing font (as I also love Open Sans) maybe put the text of the post black instead of gray would help a lot.

**Comment by MrPetovan** [comment]
On Frio the font color is a touchy subject as the accented schemes should also be considered, but I'll take any suggestions.

**Comment by Singh-in** [comment]
I created my own scheme to make the contrasts work better. Unfortunately, I could not influence the font with it. But maybe one of you has a tip.

Maybe this is a way to use custom fonts without changing them in the core.


```
<?php
/*
 * Name: loma
 * Author: one
 * 
 * List here all variables which will get overwritten through this scheme
 * Overwrites: nav_bg, nav_icon_color, link_color, background_color, contentbg_transp
 */

	$nav_bg = "#31363c";
	$nav_icon_color = "#ffffff";
	$link_color = "#515151";
	$background_color = "#f4f4f4";
	$contentbg_transp = 98;
	$font_color = '#3b3b3b';
	$font_color_darker = '#000000';
```

```
/*
    Licence    : AGPL

    Created on : 21.06.2020
    Author     : one
*/

#topbar-first, body {
    background: #191b22;
    border: 0;
}

#topbar-second, body {
    background: #f4f4f4;
    border: 0;
}

#topbar-first .dropdown.account > a, #topbar-first .dropdown.account.open > a, #topbar-first .dropdown.account > button, #topbar-first .dropdown.account.open > button, #topbar-first .dropdown.account > :hover, #topbar-first .dropdown.account.open > :hover {
    background: #191b22;
    border: 0;
}

.nav-pills .dropdown-menu, .nav-tabs .dropdown-menu, .account .dropdown-menu, .contact-photo-wrapper .dropdown-menu {
    background: #191b22;
    -webkit-box-shadow: 0px 0px 69px 4px rgba(0,0,0,0.25);
    -moz-box-shadow: 0px 0px 69px 4px rgba(0,0,0,0.25);
    box-shadow: 0px 0px 69px 4px rgba(0,0,0,0.25);
    border: 0;
}

.account .dropdown-menu li, .account .dropdown-menu li:hover {
    border: 0;
    background: #191b22;
}

#topbar-first .topbar-nav .nav-segment>a:hover {
    background: #333;
}

.account .dropdown-menu li:hover {
    background: #333 !important;
}

.account .dropdown-menu li.divider {
    background: #595754;
}

@font-face {
      font-family: 'Roboto';
      src: url('roboto.eot'); /* IE9 Compat Modes */
      src: url('roboto.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
      url('roboto.woff') format('woff'), /* Modern Browsers */
      url('roboto.ttf')  format('truetype'), /* Safari, Android, iOS */
      url('roboto.svg#svgFontName') format('svg'); /* Legacy iOS */
    }

```

**Comment by Extarys** [comment]
@Singh-in you import the font but you don't seem to apply it. This should get you started:

```
body, textarea, input, a, p {
    font-family: Roboto, Helvetica, Arial, Sans-Serif !important;
}
```

You also need to make sure `roboto.eot` exists.

**Comment by MrPetovan** [comment]
Mistakenly closed because of keyword use in #9806 description.

**Comment by Singh-in** [comment]
I have since changed the font for the entire instance.
This gives me a much clearer typeface that behaves much more pleasantly and makes the application appear more modern overall. Of course, this is always a matter of taste

view/theme/frio/css/style.css changed to 
font-family: -apple-system,".SFNSText-Regular", "San Francisco", "Roboto", "Segoe UI", "Helvetica Neue", "Lucida Grande",Helvetica,Arial,sans-serif !important;
